README.md

```diff
@@ -1,16 +1,25 @@
 ##SwiftLint for AppCode
 
-This simple plugin allows to highlight errors and warnings from SwiftLint (https://github.com/realm/SwiftLint) like this 
+This simple plugin allows to highlight errors and warnings from SwiftLint (https://github.com/realm/SwiftLint) like this:
+
 ![](http://d.pr/i/goBy/3VsjzzWL+)
 
 
 ###Installation
 
 You should already have SwiftLint installed somewhere.
- 1. Configure SwiftLint via .swiftlint.yml file, if needed
- 2. Install the plugin from the repository
+ 1. Configure SwiftLint via `.swiftlint.yml` file, if needed. See [here](https://github.com/realm/SwiftLint#configuration) fir details.
+ 
+    Note that you have to set `reporter: "xcode"`.
```

Comment:
Is this the default? If yes, this should be changed to "Note that you should not change `reporter` to a value other than `"xcode"`.